docs/kubernetes.rst

```diff
@@ -1,3 +1,13 @@
+Kubernetes Executor
+===================
+
+The kubernetes executor is introduced in Apache Airflow 1.10.0. The Kubernetes executor will create a new pod for every task instance.
+
+Example helm charts are available at `scripts/ci/kubernetes/kube/{airflow,volumes,postgres}.yaml`. The volumes are optional and depend on your configuration. There are two volumes available:
```

Comment:
We only incude scripts/ci in the _source_ distribution. I think it is nice to the use to note that.

Reply:
Ack